src/graphql/schema/uuid/user/resolvers.ts

```diff
@@ -39,11 +40,21 @@ export const resolvers: UserResolvers = {
         await dataSources.serlo.getActiveAuthorIds()
       )
     },
-    async activeDonors(_parent, _args, { dataSources }) {
-      return getUsersFromIds(
-        dataSources.serlo,
-        await activeDonorIDs(dataSources.googleSheetApi)
+    async activeDonors(_parent, { ...cursorPayload }, { dataSources }) {
```

Comment:
This can just be `cursorPayload`:
```suggestion
    async activeDonors(_parent, cursorPayload, { dataSources }) {
```